nsv13/code/modules/overmap/boarding/interiors.dm

```diff
@@ -204,6 +205,18 @@ The meat of this file. This will instance the dropship's interior in reserved sp
 
 // Anything that needs to be done after the interior loads
 /obj/structure/overmap/proc/post_load_interior()
+	if(length(ifflocs)) //Spawn the iff console at a random location!
+		var/turf/T = pick(ifflocs)
+		var/consolepath = /obj/machinery/computer/iff_console/boarding
+		if(faction == "nanotrasen" || faction == "solgov") //We use a default one for NT ships
+			consolepath = /obj/machinery/computer/iff_console
+		var/obj/C = new consolepath(T)
```

Comment:
You are not using the arg in iff console `Initialize`you added to avoid modifying the var this way.

Reply:
Yep..

Reply:
Yep..